packages/alert/index.js

```diff
@@ -0,0 +1,103 @@
+import { css, html } from 'lit';
+import { FabricElement } from '../utils';
+import { infoSvg, negativeSvg, positiveSvg, warningSvg } from './svgs';
+
+class FabricAlert extends FabricElement {
+  static properties = {
+    negative: { type: Boolean },
```

Comment:
Does not having these booleans as `reflect`ive mean the user couldn't change them?

Is that a usecase we want to support?

I'd consider a valid usecase to be a single Alert with multiple messages slotted in depending on the type of alert or message - and some of those might be warnings and some might be negative.

Reply:
> Does not having these booleans as reflective mean the user couldn't change them?
The user can change these booleans with JS by doing `alert.negative = true`, if that's what you're wondering, as long as the previous value is reset. Otherwise the original variant attribute will take precedence. E.g. `<f-alert negative />` with `alert.positive = true` will become `<f-alert negative positive />`, and that will not change the style.

According to [Lit docs](https://lit.dev/docs/components/properties/#reflected-attributes), when `reflect` is `true`, in our case the value of property "negative" will reflect to attribute "negative", and when property value changes within Alert, the attribute value would be updated respectively. 
We do not set values of Alert variants reactively within the component. I thought the idea was to control the variant/type where the alert is used, as I suggested in [pages/components/alert.html](https://github.com/fabric-ds/elements/blob/ac027c9091135fdc150dc9999879bb251aa94caa/pages/components/alert.html#L90-L109)

I did consider changing the 4 attributes to one "type" attribute but I hesitated because I saw that in other web components we relied on "types" to be separate boolean attributes, so instead of writing `<f-box type="bordered" />`  or `<f-box type="neutral" />` we'd do `<f-box bordered />` and `<f-box neutral />`.

Not sure if that made any sense :)